src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -1817,6 +1822,25 @@ export class PostgresCohortMembersService {
             const result = await this.cohortMembersRepository.save(
               cohortMemberForAcademicYear
             );
+
+            // NEW: Handle LMS enrollment for shortlisted users
+            if (result.status === 'shortlisted') {
```

ESLint

[error] Replace `'shortlisted'` with `"shortlisted"`

(prettier/prettier)